FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -336,6 +336,27 @@ public function testUseShopSearchWhenRequestIsNull()
         $this->assertTrue($result, 'Expected shop search to be triggered but FINDOLOGIC was triggered instead');
     }
 
+    public function testUseShopSearchInEmotion()
+    {
+        $request = new RequestHttp();
+        $request->setModuleName('widgets')->setControllerName('emotion')->setActionName('emotionArticleSlider');
+
+        // Create Mock object for Shopware Front Request
+        $front = $this->getMockBuilder(Front::class)
+            ->setMethods(['Request'])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $front->expects($this->atLeastOnce())
+            ->method('Request')
+            ->willReturn($request);
+
+        // Assign mocked session variable to application container
+        Shopware()->Container()->set('front', $front);
+
+        $result = StaticHelper::useShopSearch();
```

Comment:
Session variable `findologicDI` should also be set, just to be more explicit. Do the following:
* Set session variable `findologicDI` to `false` inside this test.
* Delete the session variable in method `tearDown`.